aiidalab_widgets_base/process.py

```diff
@@ -1,6 +1,8 @@
 """Widgets to work with processes."""
 # pylint: disable=no-self-use
 # Built-in imports
+from __future__ import annotations
```

Comment:
I think this is needed only for python<3.7, no?

Reply:
For the `annotations` module itself, it is deprecated on a much later version as show in https://docs.python.org/3/library/__future__.html#id1

Here I use this for the type hints such as `dict[str, str]` rather than `Dict`. I'll give it a test and see if I can drop this import.

Reply:
ok, makes sense 👍